source/AI.h

```diff
@@ -168,6 +168,7 @@ template <class Type>
 
 
 private:
+	void IssueDeploy(const PlayerInfo &player);
```

Comment:
If this method doesn't use any member variables, it should be either a static method or be implemented in the anonymous namespace.

Reply:
Moved to anonymous namespace; the class didn't have any static methods, so moving to the anonymous namespace looked nicer.